.devcontainer/devcontainer.json

```diff
@@ -19,8 +19,6 @@
     },
     // Set *default* container specific settings.json values on container create.
     "settings": {
-        // Turn off Pylance type checking since it disagrees with mypy and flags too many bogus problems:
-        "python.analysis.typeCheckingMode": "off",
```

Comment:
Since the default value for this setting is already "off", rather than re-apply the default here, I'm leaning toward assuming that users who've overridden it in their global settings should get what they asked for, rather than assuming they've just forgotten what they asked for like I had. Make sense?

Reply:
yep, that sounds like the right call